scipy/stats/morestats.py

```diff
@@ -2717,11 +2717,11 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     Parameters
     ----------
     x : array_like
-        The first set of measurements.
+        The first set of measurements, must be one-dimensional.
```

Comment:
I found this description of `x` and `y` a little confusing.  It wasn't until I read the description of the optional parameter `y` that I realized that `x` can have one of two different meanings.  Perhaps something along the lines of:
```
x : array_like
    Either the first set of measurements (in which case `y` is the second set of measurements), or 
    the differences between two sets of measurements (in which case `y` is not specified.)  
    Must be one-dimensional.
y : array_like, optional
    Either the second set of measurements (if `x` is the first set of measurements), or 
    not specified (if `x` is the differences between two sets of measurements.) 
    Must be one-dimensional.
```

The current description of `y` contains 2 consecutive COMMA SPACE sequences in 
```
The second set of measurements, , must be one-dimensional.
```